src/protect-stream.ts

```diff
@@ -202,6 +203,25 @@ export class ProtectStreamingDelegate implements CameraStreamingDelegate {
     this.controller = new this.hap.CameraController(options);
   }
 
+  private async cropSnapshot(snapshot: Buffer): Promise<Buffer> {
```

Comment:
As mentioned in my feedback in the PR - please shift this to use FFmpeg or a Node/JS-only library that doesn’t require external binaries/libraries.

Reply:
Also - I would handle exceptions within cropSnapshot rather than leaving it to the caller. My preference is to keep cropSnapshot semantics consistent with the rest of the codebase: it should either return a cropped snapshot or null.

Reply:
Done, I am now logging errors and returning null everywhere.